Bio/Seq.py

```diff
@@ -2887,6 +2915,22 @@ def lower(self):
         # sequence of the same length
         return _UndefinedSequenceData(self._length)
 
+    def isupper(self):
+        """Return True if all ASCII characters in data are uppercase.
+
+        If there are no cased characters, method returns False.
+        """
+        # Character case is irrelevant for an undefined sequence
```

Comment:
I would base the docstring on this comment. The current docstring is not really relevant for undefined sequences.

Reply:
Btw, perhaps this method should return `False` if the sequence length is 0. Or we should disallow undefined sequences of zero length.

Reply:
@mdehoon the other methods in this class have the same docstring as they do in the other classes so I was trying to stay consistent with that. No problem changing it, but it looks like it would be a departure from the style around it. Definitely want me to change it?

Reply:
OK, then let's just leave the docstrings as they are.

Reply:
Cool! Thanks for your review. Anything else I should add to the PR?